### PR TITLE
readonly mode for client

### PR DESCRIPTION
**Pull request by pkasprzyk**

### CODE REVIEW

File: neptune/new/exceptions.py [hunk 1]
```diff
@@ -316,6 +316,11 @@ def __init__(self):
         super().__init__("Cannot synchronize offline runs without a project.")
 
 
+class NeedExistingRunForReadOnlyMode(NeptuneException):
+    def __init__(self):
+        super().__init__("An existing run must be provided when running in read-only mode.")
```
Comment (aniezurawski) [inline neptune/new/exceptions.py:321]: Let's write all new exception messages in new pretty format. See `NeptuneRunResumeAndCustomIdCollision` for example.

**Review by aniezurawski — CHANGES_REQUESTED** [review]

File: neptune/new/exceptions.py (hunk 1 above)
Comment (pkasprzyk) [reply neptune/new/exceptions.py]: updated, thanks

**Review by aniezurawski — CHANGES_REQUESTED** [review]
Please add some simple test in test_client.py.

File: neptune/new/exceptions.py [hunk 2]
```diff
@@ -316,6 +316,23 @@ def __init__(self):
...
+        message = """
+    {h1}
+    ----NeedExistingRunForReadOnlyMode-----------------------------------------
+    {end}
+    Incorrect call of function {python}neptune.init(){end}.
+
+    Parameter {python}run{end} of {python}neptune.init(){end} must be provided and reference
+    an existing run when using {python}mode="read-only"{end}.
+
+    {correct}Need help?{end}-> https://docs.neptune.ai/getting-started/getting-help
```
Comment (aniezurawski) [inline neptune/new/exceptions.py:330]: @Herudaio any suggestions?

**Pull request merged**